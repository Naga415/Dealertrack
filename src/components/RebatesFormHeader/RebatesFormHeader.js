import React from "react";
import Row from "@coxautokc/fusion-ui-components/lib/Row";
import Col from "@coxautokc/fusion-ui-components/lib/Col";
import "./RebatesFormHeader.scss";

export default function RebatesFormHeader() {
  return (
    <Row id="RebatesFormHeader">
      <Col md={4} xs={4}>
        <label htmlFor="rebatesTypeLabel" id="rebates-type-left">
          Rebate Type
        </label>
      </Col>
      <Col md={4} xs={4}>
        <label htmlFor="rebatesAmountLabel" id="rebates-amount-left">
          Amount
        </label>
      </Col>
    </Row>
  );
}



