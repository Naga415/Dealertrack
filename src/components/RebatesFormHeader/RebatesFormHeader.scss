@import '~@coxautokc/fusion-theme/dist/fusion-variables.scss';

#RebatesFormHeader {
  label {
    font-weight: bold;
    min-height: 30px;
    color: #454545;
  }
}
