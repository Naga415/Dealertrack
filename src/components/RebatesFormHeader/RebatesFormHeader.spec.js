"use strict";
import React from 'react';
import renderer from 'react-test-renderer';
import RebatesFormHeader from './RebatesFormHeader';

describe('RebatesFormHeader specs', () => {
  it('renders <RebatesFormHeader />', () => {
    const component = renderer.create(<RebatesFormHeader />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
