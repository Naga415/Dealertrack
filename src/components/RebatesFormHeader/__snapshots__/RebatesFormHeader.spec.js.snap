// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RebatesFormHeader specs renders <RebatesFormHeader /> 1`] = `
<div
  className="row"
  id="RebatesFormHeader"
>
  <div
    className="col-md-4 col-xs-4"
    id={undefined}
  >
    <label
      htmlFor="rebatesTypeLabel"
      id="rebates-type-left"
    >
      Rebate Type
    </label>
  </div>
  <div
    className="col-md-4 col-xs-4"
    id={undefined}
  >
    <label
      htmlFor="rebatesAmountLabel"
      id="rebates-amount-left"
    >
      Amount
    </label>
  </div>
</div>
`;
